js_modules/dagit/packages/core/src/runs/RunDetails.tsx

```diff
@@ -116,20 +120,49 @@ export const RunConfigDialog: React.FC<{run: RunFragment; isJob: boolean}> = ({r
   const [showDialog, setShowDialog] = React.useState(false);
   const {rootServerURI} = React.useContext(AppContext);
   const runConfigYaml = yaml.stringify(run.runConfig) || '';
+
   return (
     <div>
       <Group direction="row" spacing={8}>
+        <Button
+          icon={<Icon name="edit" />}
+          onClick={() =>
+            window.open(
+              workspacePathFromRunDetails({
+                id: run.id,
+                repositoryName: run.repositoryOrigin?.repositoryName,
+                repositoryLocationName: run.repositoryOrigin?.repositoryLocationName,
+                pipelineName: run.pipelineName,
+                isJob,
+              }),
+            )
+          }
+        >
+          Open in Launchpad
+        </Button>
         <Button icon={<Icon name="tag" />} onClick={() => setShowDialog(true)}>
           View tags and config
         </Button>
-        <Tooltip content="Loadable in dagit-debug" position="bottom-right">
-          <Button
-            icon={<Icon name="download_for_offline" />}
-            onClick={() => window.open(`${rootServerURI}/download_debug/${run.runId}`)}
-          >
-            Debug file
-          </Button>
-        </Tooltip>
+        <Popover
+          position="bottom-right"
+          content={
+            <Menu>
+              <Tooltip
+                content="Loadable in dagit-debug"
+                position="bottom-right"
+                targetTagName="div"
+              >
+                <MenuItem
+                  text="Debug file"
```

Comment:
Since we have a little more space for it now, I think this text would be better as "Download debug file", otherwise "Debug" seems like the verb.